src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    this.hidden = false;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  hide() {
+    this.hidden = true;
+  };
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(victim) {
+    if (victim instanceof Herbivore
+      && victim.hidden === false
+    ) {
```

Comment:
Move condition to variable
```suggestion
    if (
      victim instanceof Herbivore
      && victim.hidden === false
    ) {
```